### DISCUSSION TITLE
Possible to migrate a discussion to an issue?

### DISCUSSION THREAD

**Discussion by zeroSteiner** [community/community#60] [Discussions]
I see in the UI it's possible to migrate an issue to a discussion, but the opposite does not seem to be exposed. Go figure one of our first discussions is a support request that belongs in the issue tracker. It'd be great if issues and discussions could be migrated both ways to keep both trackers clean.

Thanks a lot for Discussions, I look forward to using it more.

**Comment by Neilblaze** [comment]
Yes, that would be a great feature indeed!

**Comment by hardikkaurani** [comment]
I agree that supporting migration in both directions would make Discussions and Issues much easier to manage in larger repositories.

In many projects, a discussion can eventually evolve into a confirmed bug report, feature request, or actionable task. Without a migration path from Discussions to Issues, maintainers often need to manually recreate the issue, copy context, and link references, which can break continuity and waste time.

A bidirectional migration system would help by:
• Preserving the original context and conversation history
• Reducing duplicate reports
• Keeping support discussions and actionable issues properly organized
• Improving triage workflows for maintainers and contributors

It would also be useful if metadata such as labels, participants, timestamps, and linked references could optionally carry over during migration.

GitHub Discussions already works well for idea validation and community support, so having a clean transition path into Issues would make the workflow much more seamless for open source teams.

Related documentation:
https://docs.github.com/en/discussions
https://docs.github.com/en/issues

This would definitely be a valuable feature improvement.
